Sources/SwiftTryCatch/SwiftTryCatch.m

```diff
@@ -24,19 +24,14 @@ @implementation SwiftTryCatch
 /**
  Provides try catch functionality for swift by wrapping around Objective-C
  */
-+ (void)try:(__attribute__((noescape)) void(^ _Nonnull)(void))try catch:(__attribute__((noescape)) void(^ _Nonnull)(NSException *exception))catch finally:(__attribute__((noescape)) void(^ _Nullable)(void))finally;
```

Comment:
I assume we never used this functionality?

Reply:
Exactly. Wasn't used and was lowering our coverage